docs/source/models.rst

```diff
@@ -4271,172 +4271,4 @@ For more info, see:
     mlflow deployments list --help
     mlflow deployments get --help
     mlflow deployments run-local --help
-    mlflow deployments help --help
-
-
-Community Model Flavors
------------------------
-
```

Comment:
Can we keep this but make the section to be a link to the community-model-flavors.rst
 page ?

like:

```
Community Model Flavors
-----------------------

Go to `Community model flavors <link>`_ page to get ...

Reply:
Sure, I will push an update